amazon/ion/core.py

```diff
@@ -113,7 +113,19 @@ def scale_to_precision(fractional_seconds, fractional_precision):
     Returns:
         Decimal: The scaled value.
     """
-    return Decimal(fractional_seconds) * Decimal(Decimal(10) ** Decimal(-fractional_precision))
+    return Decimal(fractional_seconds).scaleb(-fractional_precision)
```

Comment:
If the parameter is a `Decimal`, why are you constructing another decimal before invoking `scaleb`?

Reply:
Sorry, parameter should be an `int`. So I cast the `int` to a Decimal and then perform the `scaleb` operation on it. Changed parameter description to be an `int` on my end.